config/skills.json

```diff
@@ -0,0 +1,991 @@
+{
```

Comment:
(For all file content)
1) Configuration of different levels of skill is mostly duplicated, consider using "base" level. (Look at spell effects as an example)
2) Using JSON arrays is highly not recommended. 
Assign arbitrary name for each effect, this will allow _mods_ to tweak individual properties of particular effect (array can only be entirely overwritten)
  "effects" : { "main":{...}, "otherEffect":{} }

Reply:
fixed.